aisynphys/database/schema/cortical_location.py

```diff
@@ -0,0 +1,44 @@
+from sqlalchemy.orm import relationship, deferred, sessionmaker, aliased
+from ... import config
+from . import make_table
+from .slice import Slice
+from .experiment import Cell, Experiment
+
+__all__ = ['CellLocation', 'CorticalSite']
+
+CellLocation = make_table(
+    name='cell_location',
+    comment='Each row holds location information for a single cell.',
+    columns=[
+        ('cell_id', 'cell.id', 'ID of the cell these locations apply to.', {'index':True}),
+        ('cortical_site_id', 'cortical_site.id', 'ID of the site location measurements fit within.', {'index':True}),
+        ('layer', 'str', 'Name of the layer the cell is in.'),
+        ('distance_to_pia', 'float', 'The distance from the cell to the pial surface in m.'),
+        ('distance_to_wm', 'float', 'The distance from the cell to the white matter in m.'),
+        ('fractional_depth', 'float', 'The cortical depth of the cell where pia is 0 and wm is 1.')
+        ])
+        
+Cell.location = relationship(CellLocation, back_populates="cell", cascade="delete", single_parent=True, uselist=False)
+CellLocation.cell = relationship(Cell, back_populates="location", single_parent=True)
+
+CorticalSite = make_table(
+    name='cortical_site',
+    comment='Each row holds measurements about one cortical site in a slice.',
+    columns=[
+        ('slice_id', 'slice.id', 'ID of the slice this site belongs to.', {'index':True}),
+        ('experiment_id', 'experiment.id', 'ID to the experiment this site is part of', {'index':True}),
+        ('pia_to_wm_distance', 'float', 'The distance (in m) from the pia to the white matter.'),
+        ('pia_position', 'object', '3D location where the pia was marked in the arbitrary coordinate system of the experiment'),
+        ('wm_position', 'object', '3D location where the wm was marked in the arbitrary coordinate system of the experiment'),
+        ('L1_L23_boundary', 'float', 'Distance from the pia to the boundary between L1 and L2/3 as a fraction of cortex.'),
```

Comment:
Layers are species- and region-specific.
Rather than columns for layers, maybe just embed a dictionary here to allow flexibility. I think it's unlikely we'll need to query on these values anyway.

Reply:
Good idea. Do you want to keep the pia and wm distance columns?

Reply:
Yeah, `pia_to_wm_distance` is good as-is.